draft-ietf-lake-edhoc.md

```diff
@@ -579,25 +579,40 @@ In order to reduce round trips and the number of messages, or to simplify proces
 
 EDHOC allows processing of external authorization data (EAD) to be defined in a separate specification, and sent in dedicated fields of the four EDHOC messages (EAD_1, EAD_2, EAD_3, EAD_4). EAD is opaque data to EDHOC.
 
-Each EAD field is a CBOR sequence (see {{CBOR}}) consisting of one or more EAD items (ead_label, ead_value) as defined below:
+Each EAD field is a CBOR sequence (see {{CBOR}}) consisting of one or more EAD items (ead_label, ? ead_value) as defined below:
 
 ~~~~~~~~~~~ CDDL
-ead = 1* (
+EAD_x = 1* ead
+
+ead = (
   ead_label : int,
-  ead_value : bstr,
+  ? ead_value : bstr,
 )
 ~~~~~~~~~~~
 
-A security application using external authorization data need to register a positive ead_label and the associated ead_value format for each EAD item it uses (see {{iana-ead}}), and describe processing and security considerations. Each application registers their own EAD items and defines associated operations. The application may define multiple uses of certain EAD items, e.g., the same EAD item may be used in different EDHOC messages with the same application.
+A security application using external authorization data needs to register a positive ead_label and an associated optional ead_value format for each EAD item it uses (see {{iana-ead}}), and specify the processing and the security considerations. Each application registers its own EAD items and defines the associated operations. The application may define multiple uses of certain EAD items, e.g., the same EAD item may be used in different EDHOC messages with the same application. Multiple occurances of an EAD item in one EAD field may also be specified.
 
-An EAD item can be either critical or non-critical, determined by the sign of the ead_label in the transported EAD item included in the EDHOC message. Using the registered positive value indicates that the EAD item is non-critical. The corresponding negative value indicates that the EAD item is critical. ead_label = 0 MUST NOT be used.
+An EAD item can be either critical or non-critical, determined by the sign of the ead_label in the transported EAD item included in the EDHOC message. Using the registered positive value indicates that the EAD item is non-critical. The corresponding negative value indicates that this EAD item is critical. ead_label = 0 is also critical and reserved for padding, see {{padding}}.
```

Comment:
Why is padding critical?

Reply:
Good catch, should be non-critical. The processing of padding is equivalent to ignoring, which is the processing of non-critical EAD that are not understood.